cmake/cmake_parse_arguments.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2007 MySQL AB, 2009 Sun Microsystems,Inc
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; version 2 of the License.
```

Comment:
https://svn.jmodelica.org/FMILibrary/trunk/Config.cmake/mergestaticlibs.cmake
can this code be included?